res/database/Default/database_ads.xml

```diff
@@ -1493,7 +1495,7 @@ Available in any well-stocked drugstore.</en>
 			</title>
 			<description>
 				<de>Die vollwertige Bio-Mahlzeit aus der Tube. Schmeckt wie bei Muttern, krümelt nicht und sooo gesund! Ideal für lange Zockerpartien.</de>
-				<en>A tube full to the brim with organic whool-food. Tastes like mom's food, does not crumble and is sooo healthy! Ideal for long gaming sessions.</en>
+				<en>The wholesome organic meal from the tube. Tastes like at mother's, does not crumble and is sooo healthy! Ideal for long gaming sessions.</en>
```

Comment:
Tastes like at home

Reply:
Changing to 'like the food at home from mother', I think a stronger association with childhood times is desired (of course you can consume tube food at home, what's so special).